packages/flutter/test/widgets/binding_test.dart

```diff
@@ -377,6 +384,49 @@ void main() {
     WidgetsBinding.instance.removeObserver(observer);
   });
 
+    testWidgets('pushRouteInformation not handled by observer returns false', (WidgetTester tester) async {
```

Comment:
Do we have test for handled cases?

Reply:
yes the existing tests on Line 275 and line 297 returns `true` for the result.  they are handled cases.

Reply:
oh i missed the diff above. Thanks